src/lsdb/loaders/hipscat/hipscat_catalog_loader.py

```diff
@@ -1,18 +1,23 @@
+from typing import List
+
 import dask.dataframe as dd
 import hipscat as hc
 import pyarrow
+from hipscat.pixel_math import HealpixPixel
+from hipscat.pixel_math.hipscat_id import HIPSCAT_ID_HEALPIX_ORDER
 
 from lsdb import io
 from lsdb.catalog.catalog import Catalog, DaskDFPixelMap
-from lsdb.core.healpix.healpix_pixel import MAXIMUM_ORDER, HealpixPixel
 from lsdb.loaders.hipscat.hipscat_loading_config import HipscatLoadingConfig
 
 
 # pylint: disable=R0903
 class HipscatCatalogLoader:
     """Loads a HiPSCat formatted Catalog"""
 
-    def __init__(self, path: str, config: HipscatLoadingConfig) -> None:
+    def __init__(
+            self, path: str, config: HipscatLoadingConfig
```

Comment:
What's your formatter doing here? Is it maybe using the default 80 line limit?

Reply:
I'm not sure, I'll have to check my config. I've changed it now though